components/kyma-environment-broker/internal/provider/azure_provider.go

```diff
@@ -98,7 +105,18 @@ func (p *AzureTrialInput) Defaults() *gqlschema.ClusterConfigInput {
 	}
 }
 
-func (p *AzureTrialInput) ApplyParameters(input *gqlschema.ClusterConfigInput, params internal.ProvisioningParametersDTO) {
+func (p *AzureTrialInput) ApplyParameters(input *gqlschema.ClusterConfigInput, pp internal.ProvisioningParameters) {
+	params := pp.Parameters
+
+	// read platform region if exists
```

Comment:
what if not exist? empty will be sent to Gardener?

Reply:
no, the default one, see method Defaults()